src/status_im/ui/components/emoji_thumbnail/color_picker.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im.ui.components.emoji-thumbnail.color-picker
+  (:require [quo.react-native :as rn]
+            [status-im.ui.components.emoji-thumbnail.styles :as styles]))
+
+(defn colors-row1 [color-circle]
+  [rn/view {:style (merge styles/emoji-picker-color-row-container styles/emoji-picker-row1-style)}
+   (for [x styles/emoji-picker-colors-row1]
+     [color-circle x])])
```

Comment:
you must use `^key` for repeating elements

Reply:
Yes, I am applying key to each item
https://github.com/status-im/status-react/pull/12594/files#diff-af8ed1cdd847ff00bd665fe97d1acf55d15538856e17c1be56b57a214f1bf778
Line: 25